comeet/UI/MyAgenda/MyAgendaViewController.swift

```diff
@@ -77,8 +76,8 @@ private extension MyAgendaViewController {
     }
     
     func goToSelectedSection() {
```

Comment:
I'm not entirely sure if this fix is correct without more context.  Could you explain what's going on here and add a method comment?